src/rev_ai/apiclient.py

```diff
@@ -814,3 +863,20 @@ def _create_job_options_payload(
 
     def _create_captions_query(self, speaker_channel):
         return '' if speaker_channel is None else '?speaker_channel={}'.format(speaker_channel)
+
+    def _build_transcript_url(self, id_, group_channels_by=None, group_channels_threshold_ms=None):
+        """Build the get transcript url.
+
+        :param id_: id of job to be requested
+        :param group_channels_by: optional, group channels by speaker or time
+        :param group_channels_threshold_ms: optional, group channels by time threshold, ms
+        :returns: url for getting the transcript
+        """
+        params = []
+        if group_channels_by is not None:
+            params.append('group_channels_by={}'.format(group_channels_by))
+        if group_channels_threshold_ms is not None:
+            params.append('group_channels_threshold_ms={}'.format(group_channels_threshold_ms))
```

Comment:
@dmtrrk i think you were saying you have doubts about this, i thiunk this is right, we are dealing with these two parameters independently